internal/server/assign_task.go

```diff
@@ -75,6 +77,10 @@ func AssignTask(client AssignTaskClient, tmpl template.Template) Handler {
 					return err
 				}
 
+				if lpa == nil && len(task.CaseItems) > 0 {
```

Comment:
Because this handler might have to deal with multiple tasks, there is always an array of them, even if you are only reassigning a single task. The loop will go through these tasks, and give `lpa` the value of the first case occurring in the case items list for the first task in the array. If we remove the nil check, it will end up with the case of the last item in the list (which might be OK anyway I suppose).